doc/source/man/daemons.rst

```diff
@@ -6,14 +6,11 @@ Daemons CLIs
     :maxdepth: 1
 
     rucio-abacus-account
-    rucio-abacus-collection-replica
```

Comment:
This exists, instead of removing, please add the man-link :-)

Reply:
Sorry, @bari12, I am not able to find these pages [here](https://github.com/rucio/rucio/tree/master/doc/source/man) [man directory of the `doc/source`]... Neither is this present anywhere else except in the `bin` directory of Rucio ... Can you please point me in the correct direction for the same?

Reply:
Yes, I meant the pages should be added. :-) The page itself is just a simple reference to parse the info from the bin/ anyway.

Reply:
Sure ... I will add them :)